--- origsrc/php-7.2.5/ext/intl/config.m4	2018-04-24 10:09:55.000000000 -0500
+++ src/php-7.2.5/ext/intl/config.m4	2018-04-25 16:05:02.823923600 -0500
@@ -85,6 +85,7 @@ if test "$PHP_INTL" != "no"; then
     breakiterator/codepointiterator_methods.cpp"
 
   PHP_REQUIRE_CXX()
+  PHP_ADD_LIBRARY(stdc++, 1, INTL_SHARED_LIBADD)
   PHP_CXX_COMPILE_STDCXX(11, mandatory, PHP_INTL_STDCXX)
   PHP_INTL_CXX_FLAGS="$INTL_COMMON_FLAGS $PHP_INTL_STDCXX $ICU_CXXFLAGS"
   case $host_alias in
