--- origsrc/php-7.3.3/build/php.m4
+++ src/php-7.3.3/build/php.m4
@@ -1155,6 +1155,7 @@ AC_DEFUN([PHP_DOES_PWRITE_WORK],[
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <stdlib.h>
 #include <unistd.h>
 #include <errno.h>
 #include <stdlib.h>
@@ -1189,6 +1190,7 @@ AC_DEFUN([PHP_DOES_PREAD_WORK],[
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <stdlib.h>
 #include <unistd.h>
 #include <errno.h>
 #include <stdlib.h>
--- origsrc/php-7.3.3/ext/iconv/config.m4	2019-03-05 08:50:42.000000000 -0500
+++ src/php-7.3.3/ext/iconv/config.m4	2019-03-29 15:39:25.329101500 -0400
@@ -30,6 +30,7 @@ if test "$PHP_ICONV" != "no"; then
       AC_MSG_CHECKING([if using GNU libiconv])
       AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #include <iconv.h>
+#include <stdio.h>
 int main() {
   printf("%d", _libiconv_version);
   return 0;
