--- origsrc/php-8.1.28/ext/iconv/config.m4
+++ src/php-8.1.28/ext/iconv/config.m4
@@ -13,9 +13,9 @@ if test "$PHP_ICONV" != "no"; then
   ])
 
   if test "$iconv_avail" != "no"; then
-    save_LDFLAGS="$LDFLAGS"
+    save_LIBS=$LIBS
     save_CFLAGS="$CFLAGS"
-    LDFLAGS="$ICONV_SHARED_LIBADD $LDFLAGS"
+	LIBS="$LIBS $ICONV_SHARED_LIBADD"
     CFLAGS="$INCLUDES $CFLAGS"
 
     AC_MSG_CHECKING([if iconv is glibc's])
@@ -144,7 +144,7 @@ int main() {
       AC_DEFINE([ICONV_BROKEN_IGNORE],0,[Whether iconv supports IGNORE])
     ])
 
-    LDFLAGS="$save_LDFLAGS"
+    LIBS=$save_LIBS
     CFLAGS="$save_CFLAGS"
 
     PHP_NEW_EXTENSION(iconv, iconv.c, $ext_shared,, [-DZEND_ENABLE_STATIC_TSRMLS_CACHE=1])
