--- origsrc/php-8.1.28/ext/opcache/jit/zend_jit.c
+++ src/php-8.1.28/ext/opcache/jit/zend_jit.c
@@ -4902,9 +4902,11 @@ ZEND_EXT_API int zend_jit_check_support(void)
 	}
 
 	if (zend_execute_ex != execute_ex) {
+/*
 		if (strcmp(sapi_module.name, "phpdbg") != 0) {
 			zend_error(E_WARNING, "JIT is incompatible with third party extensions that override zend_execute_ex(). JIT disabled.");
 		}
+*/
 		JIT_G(enabled) = 0;
 		JIT_G(on) = 0;
 		return FAILURE;
